### ISSUE TITLE
ComboBox does not support aria-label for roles with aria-role="listbox"

### ISSUE THREAD

**Issue by ibrahimrabab** [microsoft/fluentui#19394]
**Browser and OS versions**: Edge
### Describe the issue:

ComboBox as listbox form does not have aria-labels. Individual combo box components are able to add aria-label, but we need an aria-label for the entire list.
When we add an aria-label, the label exists only for the selected option that is selected from the dropdown. We need a label for the entire list when the dropdown is selected and all options are shown.

Screenshot below shows the list we want an aria-label for:
![Screenshot 2021-08-12 125942](https://user-images.githubusercontent.com/88855721/129261613-dd77bd72-535b-4482-86c5-d0b9cebd42fc.png)

The issue discovered by accessibility tool:
![Screenshot 2021-08-12 124749](https://user-images.githubusercontent.com/88855721/129261192-bdda4b49-4275-4338-a3ae-4b938109d67d.png)

Code snippet from inspection:
![Screenshot 2021-08-12 124850](https://user-images.githubusercontent.com/88855721/129261214-2db2f3a4-e580-4393-9855-5eed184f0157.png)

#### Actual behavior:

The aria label only shows up for the highlighted selection in the drop down.

#### Expected behavior:

Adding an aria label would show up as aria label for the entire list.

**Comment by gouttierre** [comment]
@ibrahimrabab  - Thanks for filing this, but you’ll need to add a CodePen repro before we can help. You can start from one of the [ComboBox ](https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox) examples (click “Export to CodePen”) or https://aka.ms/fluentpen.
